README.md

```diff
@@ -101,6 +101,29 @@ cargo-temp --worktree <branch>
 When exiting the shell (or your editor) the working tree will be cleaned up.
 Equivalent to `git worktree prune`.
 
+### Temporary Git Clone
+
+If you want to create a temporary project from a git repository, you can use the `--git` option with the repository's url:
+
+```
+cargo-temp --git <url>
+```
+
+Cargo-temp truncate the history to the last commit by default. You can change this behavior in the config file:
+
+* You can choose how many commits will stay in the history.
+  ```toml
+  git_repo_depth = 3
+  ```
+  This will leave the 3 last commits of the history.
```

Comment:
See, now you have mixed indentation. This is why I use the 4 chars rule like this:

(Feel free to ignore if you don't like)

```suggestion
 *  You can choose how many commits will stay in the history:

    ```toml
    git_repo_depth = 3
    ```

    This will leave the 3 last commits of the history.
```

(and it also looks better in text mode)